lib/devise_security_extension/rails.rb

```diff
@@ -3,10 +3,15 @@ class Engine < ::Rails::Engine
     ActiveSupport.on_load(:action_controller) do
       include DeviseSecurityExtension::Controllers::Helpers
     end
-    
-    ActionDispatch::Callbacks.to_prepare do
-      DeviseSecurityExtension::Patches.apply
-    end
 
+    if defined?(ActiveSupport::Reloader)
+      ActiveSupport::Callbacks.to_prepare do
+        DeviseSecurityExtension::Patches.apply
+      end
+    else
+      ActionDispatch::Callbacks.to_prepare do
```

Comment:
The warning I get is:

> DEPRECATION WARNING: to_prepare is deprecated and will be removed from Rails 5.1 (use ActiveSupport::Reloader.to_prepare instead)

So, should this line instead be `ActiveSupport::Reloader.to_prepare do ...` ?